developer-hub/src/polkadot_sdk/mod.rs

```diff
@@ -0,0 +1,126 @@
+//!
+//! [`frame`] is the framework used to create Substrate-based runtimes. Learn more about the
+//! distinction of a runtime and client in [`crate::reference_docs::wasm_meta_protocol`].
+//!
+//! #### Cumulus
+//!
+//! [![Cumulus-license](https://img.shields.io/badge/License-GPL3-blue)](https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/LICENSE)
+//! [![GitHub Repo](https://img.shields.io/badge/github-cumulus-white)](https://github.com/paritytech/polkadot-sdk/blob/master/cumulus)
+//!
+//! [`cumulus`] transforms FRAME-based runtimes into Polkadot-compatible parachain runtimes, and
+//! substrate-based client into Polkadot-compatible collators.
+//!
+//! #### Polkadot
+//!
+//! [![Polkadot-license](https://img.shields.io/badge/License-GPL3-blue)](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/LICENSE)
+//! [![GitHub Repo](https://img.shields.io/badge/github-polkadot-e6007a?logo=polkadot)](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot)
+//!
+//! [`polkadot`] is an implementation of a Polkadot client in Rust, by `@paritytech`. The Polkadot
+//! runtimes are located under the [`fellowship/runtime`](https://github.com/polkadot-fellows/runtimes) repository.
+//!
+//! > [`polkadot`] contains useful links to further learn about Polkadot, **but is in general not
+//! > part of the SDK**, as it is rarely used by developers who wish to build on top of Polkadot.
+//!
+//! #### XCM
+//!
+//! [![XCM-license](https://img.shields.io/badge/License-GPL3-blue)](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/LICENSE)
+//! [![GitHub Repo](https://img.shields.io/badge/github-XCM-e6007a?logo=polkadot)](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/xcm)
+//!
+//! [`xcm`], short for "cross consensus message", is the primary format that is used for
+//! communication between parachains, but is intended to be extensible to other use cases as well.
+//!
+//! ### Summary
+//!
+//! The following diagram summarizes how components of the Polkadot-SDK work together:
+#![doc = simple_mermaid::mermaid!("../../../docs/mermaid/polkadot_sdk.mmd")]
+//!
+//! 1. A Substrate-based chain is a blockchain composed of a "Runtime" and a "Client". As noted
+//!    above, the "Runtime" is the application logic of the blockchain, and the "Client" is
+//!    everything else. See [`reference_docs::wasm_meta_protocol`] for an in-depth explanation of
+//!    this. The former is built with [`frame`], and the latter is built with Substrate client
+//!    libraries.
+//! 2. Polkadot is itself a Substrate-based chain, composed of the the exact same two components.
+//!    The Polkadot client code is in [`polkadot`], and the Polkadot runtimes are controlled by the
+//!    [Polkadot Fellowship](https://github.com/polkadot-fellows).
+//! 3. A parachain is a "special" Substrate based chain, whereby both the client and the runtime
+//!    components have became "Polkadot-aware" using Cumulus.
```

Comment:
I would personally split these mermaid diagrams into 3, so we can include a description for each type of Substrate based project next to the diagram its describing.

Reply:
1896ac74b52b5302671ba82f0852819945b9c531